### PR TITLE
LRDOCS-3936 Using JNDI with Service Builder

### PR DESCRIPTION
**Pull request by ngaskill**
New troubleshooting article on Using JNDI with Service Builder

@jhinkey

### CODE REVIEW

File: develop/tutorials/articles/310-troubleshooting/14-jndi-service-builder.markdown [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+        </beans>
+
+3.  Run Service Builder. To do this via the command line for a Gradle project, 
+    navigate to your Service Builder module project's root folder and run 
+    `gradlew buildService`. Note that you can also run Service Builder from 
+    within Liferay @ide@. For more information on this, see 
+    [the tutorial on running Service Builder](/develop/tutorials/-/knowledge_base/7-0/running-service-builder-and-understanding-the-generated-code). 
+
+4.  Now you can access the JNDI data source and use it however you wish. Here's
```
Comment (sez11a) [inline develop/tutorials/articles/310-troubleshooting/14-jndi-service-builder.markdown]: There's a strange jump here. First, you configure it with Service Builder and generate services. The next logical step then should be using those services Service Builder generated. Instead, you jump to using the data source directly (i.e., not through Service Builder) and executing your own SQL queries. We should discuss this before we publish.

**Comment by jrao** [comment]
There's a BLADE samples README (which should be merged into master soon) here: https://github.com/jrao/liferay-blade-samples/blob/BLADE-178/gradle/apps/jdbc-service-builder/README.markdown

This example shows both techniques: (1) Accessing the data source directly and (2) using the service builder generated APIs. The explanation section at the end of the README explains both techniques. The troubleshooting guide article can reference the JDBC and JNDI Blade samples once they're been merged in.

For the troubleshooting guide, certainly feel free to reorder the information as seems best. There's already some duplicated content between the BLADE samples READMEs and the troubleshooting guide. I guess that's inevitable considering the troubleshooting guide topic but it's something to keep an eye on.

**Comment by sez11a** [comment]
Please see ticket (LRDOCS-3936).

**Comment by jhinkey** [comment]
@ngaskill Would you send me a PR for this branch? Thanks!

**Pull request closed without merging**